### PR TITLE
Technical Review - Temporal Reference System

### PR DESCRIPTION
**Pull request by KellyVance**
Ready for tech review.
@LukeHauck or @DavidCrosswellGSQ

### CODE REVIEW

File: vocabularies/trs.ttl [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+@prefix time: <http://www.w3.org/2006/time#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/trs>
+  a owl:Ontology ;
+  owl:imports <http://www.w3.org/2004/02/skos/core> ;
+  owl:imports <http://www.w3.org/2006/time> ;
+.
+trs:conceptScheme a skos:ConceptScheme ;
+    skos:prefLabel "Temporal Reference Systems"@en ;
```
Comment (LukeHauck) [inline vocabularies/trs.ttl]: System (singular), or Systems (plural)? 
Title of vocab is System (singular).

File: vocabularies/trs.ttl [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Chinese_calendar> ;
+  skos:definition "The traditional China calendar (officially known as the Rural Calendar [農曆; 农历; Nónglì; 'farming calendar']), or Former Calendar (舊曆; 旧历; Jiùlì), Traditional Calendar (老曆; 老历; Lǎolì) or Lunar Calendar (陰曆; 阴历; Yīnlì; 'yin calendar'), is a lunisolar calendar which reckons years, months and days according to astronomical phenomena. It is defined by GB/T 33661-2017, 'Calculation and promulgation of the Chinese calendar' issued by the Standardisation Administration of China on 12 May 2017." ;
+  skos:prefLabel "Chinese Calendar" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:GPSTime
+  a skos:Concept ;
+  a time:TRS ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Global_Positioning_System#Timekeeping> ;
+  skos:definition "GPS time is not corrected to match the rotation of the Earth, so it does not contain leap seconds or other corrections that are periodically added to UTC. GPS time was set to match UTC in 1980, but has since diverged. The lack of corrections means that GPS time remains at a constant offset with International Atomic Time (TAI) (TAI − GPS = 19 seconds)" ;
```
Comment (LukeHauck) [inline vocabularies/trs.ttl]: Ending full stop.

File: vocabularies/trs.ttl [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+  skos:prefLabel "International Atomic Time" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:IranianCalendar
+  a skos:Concept ;
+  a time:TRS ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Iranian_calendars> ;
+  skos:definition """The Iranian calendars or Iranian chronology (Persian: گاه‌شماری ایرانی‎, Gāh-Shomāri-ye Irāni) are a succession of calendars invented or used for over two millennia in Iran also known as Persia. One of the longest chronological records in human history, the Iranian calendar has been modified time and time again during its history to suit administrative, climatic, and religious purposes.
+
+As one of the few calendars designed in the era of accurate positional astronomy, the Iranian calendar uses a very complex leap year structure which makes it the most accurate solar calendar in use today.""" ;
```
Comment (LukeHauck) [inline vocabularies/trs.ttl]: Is this still part of the definition? Should be on a single line.
Also ends with many quotation marks.

File: vocabularies/trs.ttl [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+  skos:prefLabel "International Atomic Time" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:IranianCalendar
+  a skos:Concept ;
+  a time:TRS ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Iranian_calendars> ;
+  skos:definition """The Iranian calendars or Iranian chronology (Persian: گاه‌شماری ایرانی‎, Gāh-Shomāri-ye Irāni) are a succession of calendars invented or used for over two millennia in Iran also known as Persia. One of the longest chronological records in human history, the Iranian calendar has been modified time and time again during its history to suit administrative, climatic, and religious purposes.
```
Comment (LukeHauck) [inline vocabularies/trs.ttl]: Definition begins with multiple quotation marks.

File: vocabularies/trs.ttl [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Julian_calendar> ;
+  skos:definition "The Julian calendar, proposed by Julius Caesar in 708 AUC (46 BC/BCE), was a reform of the Roman calendar. It took effect on 1 January 709 AUC (45 BC/BCE), by edict. It was designed with the aid of Greek mathematicians and Greek astronomers such as Sosigenes of Alexandria. " ;
+  skos:prefLabel "Julian Calendar" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:JulianDay
+  a skos:Concept ;
+  a time:TRS ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Julian_day> ;
+  skos:definition """Julian day is the continuous count of days since the beginning of the Julian Period and is used primarily by astronomers, and in software for easily calculating elapsed days between two events (e.g. food production date and sell by date).
```
Comment (LukeHauck) [inline vocabularies/trs.ttl]: Multiple quotation marks beginning and end the definition.

File: vocabularies/trs.ttl [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+The Julian Day Number (JDN) is the integer assigned to a whole solar day in the Julian day count starting from noon Universal time, with Julian day number 0 assigned to the day starting at noon on Monday, January 1, 4713 BC, proleptic Julian calendar (November 24, 4714 BC, in the proleptic Gregorian calendar), a date at which three multi-year cycles started (which are: Indiction, Solar, and Lunar cycles) and which preceded any dates in recorded history. For example, the Julian day number for the day starting at 12:00 UT on January 1, 2000, was 2 451 545. """ ;
```
Comment (LukeHauck) [inline vocabularies/trs.ttl]: Is this still part of the definition? Should be on a single line.
Also ends with many quotation marks.

File: vocabularies/trs.ttl [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+  skos:definition "The Maya calendar is a system of calendars used in pre-Columbian Mesoamerica and in many modern communities in the Guatemalan highlands, Veracruz, Oaxaca and Chiapas, Mexico." ;
+  skos:prefLabel "Maya Calendar" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:ParliamentsOfAustraliaTimeline
+  a skos:Concept ;
+  a time:TRS ;
+  dct:source <https://en.wikipedia.org/wiki/Chronology_of_Australian_federal_parliaments> ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Parliament_of_Australia> ;
+  skos:definition """The Government of Australia is the government of the Commonwealth of Australia, a federal parliamentary constitutional monarchy. It is also commonly referred to as the Australian Government, the Commonwealth Government, Her Majesty's Government, or the Federal Government.
```
Comment (LukeHauck) [inline vocabularies/trs.ttl]: Multiple quotation marks.

File: vocabularies/trs.ttl [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+
+The Parliament of Australia (officially the Federal Parliament, also called the Commonwealth Parliament) is the legislative branch of the government of Australia. It consists of three elements: the Crown (represented by the Governor-General), the Senate and the House of Representatives. The combination of two elected chambers, in which the members of the Senate represent the states and territories while the members of the House represent electoral divisions according to population, is modelled on the United States Congress. Through both chambers, however, there is a fused executive, drawn from the Westminster system.
+
+This timeline of Parliaments of Australia starts in 1901 (Gregorian Caledar) when Australia was formed as an country from pre-existing colonies that where Dominions of the British Empire and were not sovereign states themselves. The 1901 Australian federal election for the inaugural Parliament of Australia was held in Australia on Friday 29 March and Saturday 30 March 1901 after which Edmond Barton was sworn in as the inaugural Prime Minister.
+
+Epoch within this timeline are numbered 1st, 2nd etc. up to (as of October, 2019, Gegorian Calendar) the 44th.""" ;
```
Comment (LukeHauck) [inline vocabularies/trs.ttl]: Is this still part of the definition? Should be on a single line.
Also ends with many quotation marks.

File: vocabularies/trs.ttl [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+  skos:definition "Unix time (also known as Epoch time, POSIX time, seconds since the Epoch, or UNIX Epoch time) is a system for describing a point in time. It is the number of seconds that have elapsed since the Unix epoch, that is the time 00:00:00 UTC on 1 January 1970, minus leap seconds." ;
+  skos:prefLabel "UNIX Time" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:UTCTime
+  a skos:Concept ;
+  a time:TRS ;
+  dct:source <https://www.bipm.org/en/bipm-services/timescales/> ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Unix_time> ;
+  skos:definition "The atomic time scale that forms the basis for the coordinated dissemination of standard frequencies and time signals" ;
```
Comment (LukeHauck) [inline vocabularies/trs.ttl]: Ending full stop.

File: vocabularies/trs.ttl [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+@prefix trs: <http://linked.data.gov.au/def/trs#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix time: <http://www.w3.org/2006/time#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/trs>
```
Comment (LukeHauck) [inline vocabularies/trs.ttl]: Should "trs" be spelled out? **temporal-reference-system**

File: vocabularies/trs.ttl (hunk 1 above)
Comment (KellyVance) [reply vocabularies/trs.ttl]: @nicholascar as the original submitter of this vocabulary into the GSQ, can you please comment on the feedback from Luke and/or correct the file where necessary?

File: vocabularies/trs.ttl (hunk 3 above)
Comment (KellyVance) [reply vocabularies/trs.ttl]: I believe the triple quote marks at the start and end of definitions allow for non-standard characters and quote a marks to exist within the definition without creating errors.

Comment (LukeHauck) [reply vocabularies/trs.ttl]: Thanks for clearing that up Vance!

File: vocabularies/trs.ttl (hunk 4 above)
Comment (LukeHauck) [reply vocabularies/trs.ttl]: Resolved in comment above.

File: vocabularies/trs.ttl (hunk 7 above)
Comment (LukeHauck) [reply vocabularies/trs.ttl]: Resolved in comment above.

File: vocabularies/trs.ttl (hunk 5 above)
Comment (LukeHauck) [reply vocabularies/trs.ttl]: Resolved in comment above.

File: vocabularies/trs.ttl [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:IranianCalendar
+  a skos:Concept ;
+  a time:TRS ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Iranian_calendars> ;
+  skos:definition """The Iranian calendars or Iranian chronology (Persian: گاه‌شماری ایرانی‎, Gāh-Shomāri-ye Irāni) are a succession of calendars invented or used for over two millennia in Iran also known as Persia. One of the longest chronological records in human history, the Iranian calendar has been modified time and time again during its history to suit administrative, climatic, and religious purposes.
+
+As one of the few calendars designed in the era of accurate positional astronomy, the Iranian calendar uses a very complex leap year structure which makes it the most accurate solar calendar in use today.""" ;
+  skos:prefLabel "Iraian Calendar" ;
```
Comment (LukeHauck) [inline vocabularies/trs.ttl]: "Iranian"

File: vocabularies/trs.ttl (hunk 1 above)
Comment (nicholascar) [reply vocabularies/trs.ttl]: The Concept Scheme should be plural: it contains a number of Temporal Reference System instance. See the Concept Scheme label for [Data Access Rights](https://github.com/geological-survey-of-queensland/vocabularies/blob/master/vocabularies/data-access-rights.ttl) which is also plural.

File: vocabularies/trs.ttl (hunk 2 above)
Comment (nicholascar) [reply vocabularies/trs.ttl]: Added: https://github.com/geological-survey-of-queensland/vocabularies/pull/58/commits/35ef42f4372569467da79a827ce8f9096c564189

File: vocabularies/trs.ttl (hunk 6 above)
Comment (nicholascar) [reply vocabularies/trs.ttl]: It is part of the definition but the definition's enclosed in triple quotes """ which, in Turtle syntax means line breaks are respected. See the Turtle spec's section on [quoted literals](https://www.w3.org/TR/turtle/#turtle-literals).

File: vocabularies/trs.ttl (hunk 8 above)
Comment (nicholascar) [reply vocabularies/trs.ttl]: as above

File: vocabularies/trs.ttl (hunk 9 above)
Comment (nicholascar) [reply vocabularies/trs.ttl]: added as per https://github.com/geological-survey-of-queensland/vocabularies/commit/c5c5bff5fa2e820aa68de308a74d97089b27a708

File: vocabularies/trs.ttl (hunk 10 above)
Comment (nicholascar) [reply vocabularies/trs.ttl]: I don't think so: no need within the identifier (the URI) as it's everywhere in the metadata.

File: vocabularies/trs.ttl (hunk 3 above)
Comment (nicholascar) [reply vocabularies/trs.ttl]: Not really for "non-standard characters", more line breaks, see the formal definition: [turtle-literals](https://www.w3.org/TR/turtle/#turtle-literals)

File: vocabularies/trs.ttl (hunk 11 above)
Comment (nicholascar) [reply vocabularies/trs.ttl]: Fixed in https://github.com/geological-survey-of-queensland/vocabularies/commit/f6282d61b401996386c31bd4d156ba02c271f5b4

**Comment by nicholascar** [comment]
@LukeHauck @KellyVance Thanks for the typos review! I think I've addressed everything.

File: vocabularies/trs.ttl (hunk 1 above)
Comment (KellyVance) [reply vocabularies/trs.ttl]: All concept schemes have multiple concepts or they are redundant. In several of our vocabularies we refer to the name as singular. i.e. the selected term from the vocabulary is the Earth Science Data Category, the Sampling Method, the Queensland Resource Permit. Using the GeoSciML as a reference the vocabularies are all singular. http://resource.geosciml.org/def/voc/
It does raise the issue that one way or another these should be consistent throughout all our vocabularies.

**Pull request merged**